### ISSUE TITLE
Insufficient color contrast on file delete modal button

### ISSUE THREAD

**Issue by sapegin** [CompuIves/codesandbox-client#1702]
# 🐛 bug report

## Description of the problem

Text on cancel button is unreadable. Also both buttons have no focus styles.

![image](https://user-images.githubusercontent.com/70067/55475369-139e8000-5614-11e9-9f88-9f46b3b012cd.png)

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  |  PROD-1554195707-bfa032269 |
| Browser | Chrome |
| Operating System | macOS |
